pylasu/parsing/results.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+from typing import List
+
+from antlr4 import ParserRuleContext, Token
+
+from pylasu.model import Source
+from pylasu.validation.validation import WithIssues, IssueType, Issue
+
+
+@dataclass
+class FirstStageResult(WithIssues):
```

Comment:
`FirstStageResult` should have an `issues: List[Issue] = default_factory(list)` property to avoid a `dataclass` missing field error.

Reply:
Note that the `issues` list here would not be required if following the comment concerning `WithIssues` below